wandb/apis/public.py

```diff
@@ -2605,8 +2628,17 @@ def get(
         }
         variables.update(kwargs)
 
-        response = client.execute(query, variable_values=variables)
-        if response.get("project") is None:
+        response = None
+        try:
+            response = client.execute(query, variable_values=variables)
+        except Exception:
```

Comment:
it feels not great to fall back on any exception, but I think this makes sense. We just end up falling back to the original query and if the exception was unrelated to the original query structure, the next query would just raise it.

But we should maybe have a TODO to find a better way to do this in the future